CMakeLists.txt

```diff
@@ -182,6 +182,7 @@ add_library(redisraft MODULE
         src/fsync.c
         src/join.c
         src/log.c
+        src/log_utils.c
```

Comment:
We don't need this file anymore (and also log_utils.h).

Reply:
see a booboo, missed one thing.  fixing